components/svg/UniqueIconComposer.jsx

```diff
@@ -1,5 +1,4 @@
 import { IconComposer } from "@rubin-epo/epo-react-lib";
-import svgShape from "@/shapes/svg";
```

Comment:
I think the svgShape only applies to svgs, not the composer.  The Proptypes for the composer would be more freeform and/or would be like the icon component string, not the svg porps?